docs/guide/test-fixtures.md

```diff
@@ -104,6 +104,37 @@ In the above, we have shown how to define a fixture about a DB table. To define
 (e.g. a fixture about certain files and directories), you may extend from the more general base class
 [[yii\test\Fixture]] and override the [[yii\test\Fixture::load()|load()]] and [[yii\test\Fixture::unload()|unload()]] methods.
 
+In case of many to many relations it is possible to load fixtures for related models
+using the `dataExtra` property, like the following:
+
+```php
+namespace app\tests\fixtures;
+
+use yii\test\ActiveFixture;
+
+class UserFixture extends ActiveFixture
+{
+    public $modelClass = 'app\models\User';
+
+    public $dataExtra = [
```

Comment:
Since we're defining fixtures for relations here, it's better to name it `$relationFixtures` instead of general `$dataExtra`. Do you agree?